client/extensions/woocommerce/app/promotions/helpers.js

```diff
@@ -0,0 +1,26 @@
+
+export function isValidPromotion( promotion ) {
```

Comment:
Not a blocker but might be nice to add some tests around this in a follow-up PR.

Reply:
I'll add it to my post-launch list. Thanks.